README.md

```diff
@@ -0,0 +1,34 @@
+# Wicked Migration
+
+Provides a webserver for migrating Wicked configuration files to NetworkManager configuration files.
+
+### Build:
+
+`cargo build --release`
+
+### Run Server:
+
+Options
+| Option | Description |
+|--------|-------------|
+| -s     | Specify where static files are located |
+| -i     | Set IP address |
+| -p     | Set port |
+| -d     | Set location for the database |
+| -h     | Show help information |
+
+Configuration files are automatically deleted after 5 minutes or upon retrieval.
+
+### Migration:
+
+Migration can be accessed through a web frontend or the commandline:
+
+#### Commandline:
+
+multiple files:
```

Comment:
You say `multiple files` but provide the command for the single file endpoint. Also I would say an example for both the single file and multiple files endpoint are useful.